Include the Debian menu. Else all would end up under lost & found.

Patch by Sune Vuorela.

--- a/kded/applications.menu
+++ b/kded/applications.menu
@@ -437,5 +437,12 @@
 		</And>
 	</Include>
 	<DefaultMergeDirs/>
+
+	<Menu>
+		<Name>Debian</Name>
+		<Directory>debian.directory</Directory>
+		<MergeFile>debian-menu.menu</MergeFile>
+	</Menu>
+
 	<MergeFile>applications-kmenuedit.menu</MergeFile>
 </Menu>
